helm_deploy/hmpps-restricted-patients-api/values.yaml

```diff
@@ -24,6 +23,10 @@ generic-service:
         deny all;
         return 401;
       }
+      location /process-past-date-restricted-patients {
```

Comment:
intended to be called by cron job running locally so don't want exposed externally